app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] <b>REQUIRED:</b> The same thing goes here. For `Post`, `Like` and `Comment` Objects, We will be addressing the user who those belongs to as an `author`. and to do so, we have to define it here. Kindly do that.

```ruby
belongs_to :author, class_name: 'User'

Reply:
This change has been done perfectly